composer.json

```diff
@@ -25,7 +25,8 @@
 		"pelago/emogrifier": "@dev",
 		"greeny/nette-slack-logger": "^1.2",
 		"webuni/srazy-api-client": "1.0.x-dev",
-		"dms/meetup-api-client": "2.0.x-dev"
+		"dms/meetup-api-client": "2.0.x-dev",
+		"drahak/restful": "@dev"
```

Comment:
I think this package is dead for few years.

I recommend checking this fresh package: https://github.com/Odbav-to/presenter-route#rest-routes